queries.sql

```diff
@@ -29,4 +29,109 @@ WHERE
 
 SELECT * from animals
 WHERE
-    weight_kg BETWEEN 10.4 and 17.3;
\ No newline at end of file
+    weight_kg BETWEEN 10.4 and 17.3;
+
+-- Inside a transaction update the animals table by setting the species column 
+-- to unspecified. Verify that change was made. Then roll back the change and 
+-- verify that the species columns went back to the state before the transaction.
+
+BEGIN;
+UPDATE animals
+SET species = 'unspecified';
+
+SELECT * from animals
+
+ROLLBACK;
+
+SELECT * from animals
+
+-- Inside a transaction:
+-- Update the animals table by setting the species column to digimon for all 
+-- animals that have a name ending in mon.
+-- Update the animals table by setting the species column to pokemon for all 
+-- animals that don't have species already set.
+-- Verify that changes were made.
+-- Commit the transaction.
+-- Verify that changes persist after commit.
+-- Take a screenshot of the results of your actions.
+
+BEGIN;
+UPDATE animals
+SET species = 'digimon'
+WHERE name like '%mon%';
+
+UPDATE animals
+SET species = 'pokemon'
+WHERE species IS NULL;
+
+SELECT * from animals;
+
+COMMIT;
+
+SELECT * from animals
+
+BEGIN;
+DELETE from animals;
+
+SELECT * from animals;
+
+ROLLBACK;
+
+SELECT * from animals
+
+-- Inside a transaction:
+-- Delete all animals born after Jan 1st, 2022.
+-- Create a savepoint for the transaction.
+-- Update all animals' weight to be their weight multiplied by -1.
+-- Rollback to the savepoint
+-- Update all animals' weights that are negative to be their weight 
+-- multiplied by -1.
+-- Commit transaction
+-- Take a screenshot of the results of your actions.
+
+BEGIN;
+DELETE FROM animals 
+WHERE date_of_birth > '2022-01-01';
+
+SELECT * from animals;
+
+SAVEPOINT sp1;
+UPDATE animals
+SET weight_kg = weight_kg * (-1);
+
+SELECT * from animals;
+
+ROLLBACK TO SAVEPOINT sp1;
+
+UPDATE animals
+SET weight_kg = weight_kg* (-1)
+WHERE weight_kg < 0;
+
+SELECT * from animals;
+
+Commit;
+
+SELECT * from animals
+
+-- ////Picture 4///
+
+SELECT count(*) from animals;
+
+SELECT count(*) from animals
+WHERE escape_attempts = 0;
+
+SELECT avg(weight_kg) from animals;
+
+SELECT neutered, SUM(escape_attempts) AS total_escapes
+	FROM animals
+	GROUP BY neutered
+	ORDER BY total_escapes DESC
+	LIMIT 1;
+
```

Comment:
**[OPTIONAL BUT HIGHLY RECOMMENDED]**

- [ ] Great work so far but I guess you misunderstood the requirement because you are supposed to get that individual animal who escapes the most from the neutered and non-neutered animals 🤔 so you should use `AVG` or `MAX` instead of SUM because SUM is doing summation of all the escape attempts which we don't want 😨